tests/TestCase/Mailer/Transport/QueueTransportTest.php

```diff
@@ -49,9 +49,9 @@ public function testSendWithEmail() {
 
 		$result = $this->QueueTransport->send($message);
 		$this->assertSame('Queue.Email', $result['job_task']);
-		$this->assertTrue(strlen($result['data']) < 10000);
+		$this->assertNotEmpty($result['data']);
```

Comment:
I wonder if there is value in also asserting some items in the array are as expected, so the test can confirm data has passed through the code correctly now that the Serializers have been removed.

Although, it could potentially make the test a little fragile